src/downgrader.php

```diff
@@ -58,7 +58,8 @@ public function handle_submit() {
 			return;
 		}
 
-		$target_version = \filter_var( \wp_unslash( $_POST['target_version'] ) );
+		$target_version = ( isset( $_POST['target_version'] ) ) ? \sanitize_text_field( \wp_unslash( $_POST['target_version'] ) ) : null;
```

Comment:
I think you want to add a `is_string` check here too. I'm not sure unslash and sanitize text field like operating on non-strings.

Also, the `null` fallback does not seem compatible with the `downgrade` method! What did you want to achieve?